wifi.go

```diff
@@ -0,0 +1,75 @@
+package keylight
+
+import (
+	"bytes"
+	"context"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+// A WiFiInfo contains the information neccessary for a Key Light to connect
+// to a wireless network.
+type WiFiInfo struct {
+	SSID         string       `json:"ssid,omitempty"`
+	Passphrase   string       `json:"passphrase,omitempty"`
+	SecurityType WiFiSecurity `json:"securityType,omitempty"`
+}
+
+// WiFiSecurity is the security type of the wireless network.
+//
+// The Elgato Key Light supports none, WEP, and WPA/WPA2 Personal
+type WiFiSecurity int
+
+const (
+	None WiFiSecurity = 0
+	WEP  WiFiSecurity = 1
+	WPA  WiFiSecurity = 2
+)
+
+// SetWiFiInfo encrypts and updates a Key Light's WiFi configuration.
```

Comment:
I think the "encrypts and" part can be removed since it's probably an implementation detail.